atd-vze/src/views/Crashes/Recommendations.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+import { useQuery, useMutation } from "@apollo/react-hooks";
+import { Card, CardHeader, CardBody, CardFooter, Table, Button } from "reactstrap";
+import { recommendationsDataMap } from "./recommendationsDataMap";
+import { GET_RECOMMENDATIONS } from "../../queries/recommendations";
+import { useAuth0, isReadOnly } from "../../auth/authContext";
+
+// declare fatality review board recommendations component
+const Recommendations = ({ crashId }) => {
+
+  // fetch data from database using graphQL query
+  const { loading, error, data, refetch } = useQuery(GET_RECOMMENDATIONS, {
+    variables: { crashId },
+  });
+
+  if (loading) return "Loading...";
+  if (error) return `Error! ${error.message}`;
+
+  // if crash has data in recommendations table, display it
+  const displayData = (field) => {
```

Comment:
this function is being constructed on every render. it's a teen tiny performance hit, but you could wrap it in a `useCallback` hook to optimize it ([docs](https://reactjs.org/docs/hooks-reference.html#usecallback)).

Reply:
oo i havent heard of `useCallback` before, ill look into that